ur_robot_driver/src/hardware_interface.cpp

```diff
@@ -1185,23 +1178,24 @@ void HardwareInterface::cancelInterpolation()
 
 void HardwareInterface::passthroughTrajectoryDoneCb(urcl::control::TrajectoryResult result)
 {
-  // This should be forwarded to the passthrough controller. Probably there's a better way than
-  // using a member function, this is just an example of registering a callback for the trajectory
-  // end.
+  hardware_interface::ExecutionState final_state;
   switch (result)
   {
     case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_SUCCESS:
     {
+      final_state = hardware_interface::ExecutionState::SUCCESS;
       ROS_INFO_STREAM("Forwarded trajectory finished successful.");
       break;
     }
     case urcl::control::TrajectoryResult::TRAJECTORY_RESULT_CANCELED:
     {
+      final_state = hardware_interface::ExecutionState::SUCCESS;
```

Comment:
```suggestion
      final_state = hardware_interface::ExecutionState::PREEMPTED;
```

Reply:
Ah, this is where this came from :-D fixed in b59a729